docs/DeveloperGuide.md

```diff
@@ -453,23 +460,125 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Adding a friend
+1. Adding a friend to gitGud
 
-1. Deleting a person while all persons are being shown
+    1. Prerequisites: List all friends using the `friend --list` command. There should not be a friend with 
+       `FRIEND_ID` Draco already stored in gitGud.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    2. Test case: `friend --add Draco --name Marcus`<br>
+       Expected: Friend with `FRIEND_ID` Draco is added. gitGud states that `FRIEND_ID` is added.
+       
+    3. Test case: `friend --add MrFeely`<br>
+       Expected: Friend with `FRIEND_ID` MrFeely is added. gitGud states that `FRIEND_ID` is added.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`, ...`
+       Expected: Similar to previous.
 
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+2. _{ more test cases …​ }_
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+### Adding a game
+1. Adding a game to gitGud
+
+    1. Prerequisites: List all persons using the `list --game` command. There should not be a game with `GAME_NAME` 
+       Valorant already storied in gitGud.
+
+    2. Test case: `game --add Valorant`<br>
+       Expected: Friend with `FRIEND_ID` Draco is added. gitGud states that `FRIEND_ID` is deleted.
+
+    3. Test case: `game --add`<br>
+       Expected: No person is added. Error details shown in the status message. Status bar remains the same.
```

Comment:
This should be 'No game is added' instead of No person is added since its referring to adding a game to gitGud